lochness/rpms/__init__.py

```diff
@@ -293,9 +294,24 @@ def initialize_metadata(Lochness: 'Lochness object',
 
                 # Consent date
                 if rpms_consent_colname in df_measure.index:
-                    subject_dict['Consent'] = datetime.strptime(
+                    consent_tmp = datetime.strptime(
                             df_measure[rpms_consent_colname],
-                            '%d/%m/%Y %I:%M:%S %p').strftime('%Y-%m-%d')
```

Comment:
The hard work @kcho proposed is probably not necessary. The `df_measure` can be sorted and its first row can be retained:

```python
if rpms_consent_colname in df_measure.index:
    _df_measure= df_measure.sort_values('interview_date',
        key= lambda dates: [datetime.striptime(x, '%m/%d/%Y')
            for x in dates]).iloc[0]
    subject_dict['Consent']=datetime.strptime(
        _df_measure['interview_date'], '%m/%d/%Y').strftime('%Y-%m-%d')
```

Reply:
Tashrif already does it this way but he uses `iloc[-1]` for cohort determination:
https://github.com/AMP-SCZ/utility/blob/1ce284d85e237447047b16cb5a6787951ee5de6e/rpms_to_redcap.py#L184-L187